### PR TITLE
Prep for container use

### PR DESCRIPTION
**Pull request by rjschwei**
This is the start of being able to use cloud-regionsrv-client from a container. Inside a container we do not want systemd and it's dependencies. Change the macro used such that systemd is not automatically set up as package dependency on SLE 15 and later distributions.

### CODE REVIEW

File: cloud-regionsrv-client.spec [hunk 1]
```diff
@@ -45,7 +45,11 @@ Requires:       regionsrv-certs
 Requires:       zypper
 BuildRequires:  systemd
 Conflicts:      container-suseconnect
+%if 0%{?suse_version} == 1315
 %{?systemd_requires}
+%else
+%{?systemd_oredering}
```
Comment (jgleissner) [inline cloud-regionsrv-client.spec:51]: Shouldn't that be `systemd_ordering`?

**Review by jgleissner — CHANGES_REQUESTED** [review]
The spec should probably contain `BuildRequires: systemd-rpm-macros`.

**Comment by jgleissner** [comment]
> The spec should probably contain `BuildRequires: systemd-rpm-macros`.

It appears rpm already requires this so I guess adding this wasn't really necessary (sorry), but obviously won't harm.

**Comment by rjschwei** [comment]
It's always there as was at one point the python-rpm-macros until it was not and then created a bunch of failures. Good to be explicit.

**Pull request merged**